kazoo/recipe/queue.py

```diff
@@ -262,6 +262,24 @@ def consume(self):
         else:
             return False
 
+    def release(self):
+        """Removes the lock from currently processed item without consuming it.
+
+        :returns: True if the lock was removed successfully, False otherwise.
+        :rtype: bool
+        """
+        if not self.processing_element is None and self.holds_lock:
+            id_, value = self.processing_element
+            with self.client.transaction() as transaction:
+                transaction.delete("{path}/{id}".format(
```

Comment:
It seems like u could just use https://github.com/python-zk/kazoo/blob/master/kazoo/protocol/paths.py#L19 but a minor nit...